extensions/amp-date-display/0.2/date-display.js

```diff
@@ -15,7 +15,8 @@
  */
 
 import {useResourcesNotify} from '../../../src/preact/utils';
-import {userAssert} from '../../../src/log';
+// TODO(wassgha): Discuss a replacement for userAssert
+// import {userAssert} from '../../../src/log';
```

Comment:
@dvdyakonov / @jridgewell would love to hear your input on this

Reply:
I think we should figure out a way to report warnings maybe, but never assert in React-related code.

@wassgha Any patterns we see out there for warnings in component libraries?

@jridgewell WDYT, if we had a special static function: `warnComponent(fatal, message)` that gets rewritten to `userAssert` and `userWarn` in AMP mode and possibly completely striped in React mode or replaced with `console.error` or such?

Reply:
Also, @wassgha, does storybook point the way here? Do they display component errors/warnings in one of the tabs?

Reply:
Just use `console.warn` and `console.error`? We can rewrite it.